src/config/params.rs

```diff
@@ -140,3 +140,17 @@ fn color_to_solid_source(x: Color) -> SolidSource {
     let bytes = x.to_be_bytes();
     SolidSource::from_unpremultiplied_argb(bytes[3], bytes[0], bytes[1], bytes[2])
 }
+
+fn parse_cent(input: Option<&String>, default: u32) -> u32 {
+    let input: String = match input {
+        Some(a) => a.split_whitespace().collect(),
+        None => return default,
+    };
+    if input.contains('%') {
+        return match input.trim().strip_suffix('%').unwrap().parse::<u32>() {
+            Ok(a) => (a * default) / 100,
```

Comment:
Unfortunately that won't work, current value is interpreted as pixels. Thus `80%` will be converted to 80px (instead of expected `0.8 * screen_side`).

Instead I'll suggest to create a separate type for width/height, like this:

```rust
enum SideSize {
    Absolute { pixels: u32 },
    Relative { ratio: f32 },
}
```

Then at the surface.rs `SideSize::Relative` should be converted into the absolute value.

Reply:
I was thinking relative to the default values,  So 80% would be converted to 320px for height.
I seem to have missed the intent. 
I will try this new way.

Reply:
Indeed, some of issues are vaguely described, mostly because these are written for myself. Sorry for inconvenience, feel free to ask any question, if anything else is ambiguous.